### ISSUE TITLE
READ: No need for game IDs anymore

### ISSUE THREAD

**Issue by fmartingr** [RenanGreca/Switch-Screenshots#46]
Hey folks, I, like some of you, like to have backup of all my systems screenshots and maintaining a list of game IDs for the switch has been an awful experience as of today.

Just you everyone know, Nintendo Switch 11.0+ introduced a way of copying screenshots directly from the Switch by connecting it via an USB cable and it displays screenshots **in folders with the game's localized name**:

![Screenshot 2022-03-13 at 16 19 28](https://user-images.githubusercontent.com/812088/158066633-817be8e7-33af-48ba-a6c3-f7ad7c7c80ec.png)

You can get more information here: https://en-americas-support.nintendo.com/app/answers/detail/a_id/53664/~/how-to-transfer-screenshots-and-video-captures-to-a-computer-via-a-usb-cable

**Comment by RenanGreca** [comment]
Hi, thank you so much for the information! It's good to finally have a practical way of doing this.

**Comment by RenanGreca** [comment]
@fmartingr Hey, did you have to do anything specific to get this to work? I tried using Android File Manager as your screenshot indicates, but it just says "Unable to access device storage".

**Comment by fmartingr** [comment]
@RenanGreca You need to go to system settings and enable the USB connection on the Switch (under Data Management > Screenshots IIRC). In my case enabling that launches Android File Manager automatically on macOS.

**Comment by RenanGreca** [comment]
Yeah, I did that, I don't know why it didn't work... I'll keep looking.

**Comment by fmartingr** [comment]
> Yeah, I did that, I don't know why it didn't work... I'll keep looking.

Try with different cables, you may be using a power only cable.

**Comment by gamebits** [comment]
My Switch confirmed it was connected to my Ventura 13.6.3 Mac (so the correct cable was indeed being used), but neither [Android File Transfer](https://www.android.com/filetransfer/) nor [MacDroid](https://www.macdroid.app/) recognized the Switch.

This repo's Python script still ended up being the most viable solution for me. YMMV!
